internal/cli/auth/logout.go

```diff
@@ -86,7 +86,7 @@ func LogoutBuilder() *cobra.Command {
 				return err
 			}
 			opts.Entry = s
-			if err := opts.PromptWithMessage("Are you sure you want to log out of account %s?"); err != nil {
+			if err := opts.PromptWithMessage("Are you sure you want to log out of account %s?"); err != nil || !opts.Confirm {
```

Comment:
what about not even asking `Are you sure you want to log out of account` is opts.Confirm is `true`?
if !opts.Confirm {
    if err := opts.PromptWithMessage("Are you sure you want to log out of account %s?"); err != nil {
        return err
    }
}